### ISSUE TITLE
"Display large" letter spacing should be 0.0 but is -0.25

### ISSUE THREAD

**Issue by anthonymoretti** [flutter/flutter#120876]
## Steps to Reproduce

1. See [line 768 in "typography.dart"](https://github.com/flutter/flutter/blob/9944297138/packages/flutter/lib/src/material/typography.dart#L768).

**Expected results:** 
`letterSpacing: 0.0`

**Actual results:** 
`letterSpacing: -0.25`

<details>
<summary>Code sample</summary>

See [line 768 in "typography.dart"](https://github.com/flutter/flutter/blob/9944297138/packages/flutter/lib/src/material/typography.dart#L768).

</details>

<details>
  <summary>Logs</summary>

```
[flutter] flutter doctor -v
[✓] Flutter (Channel stable, 3.7.3, on macOS 13.2.1 22D68 darwin-x64, locale en-US)
    • Flutter version 3.7.3 on channel stable at /Users/anthonymoretti/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 9944297138 (8 days ago), 2023-02-08 15:46:04 -0800
    • Engine revision 248290d6d5
    • Dart version 2.19.2
    • DevTools version 2.20.1

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14C18
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] VS Code (version 1.75.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.58.0

[✓] Connected device (3 available)
    • iPhone 14 Pro (mobile) • ED506CAA-EB76-47D8-BBDC-751CB0EE7EC8 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-16-2 (simulator)
    • macOS (desktop)        • macos                                • darwin-x64     • macOS 13.2.1 22D68 darwin-x64
    • Chrome (web)           • chrome                               • web-javascript • Google Chrome 110.0.5481.100

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
exit code 0

```

</details>

**Comment by danagbemava-nc** [comment]
Hi @anthonymoretti, Thanks for the report. 

![Screenshot 2023-02-16 at 15 04 29](https://user-images.githubusercontent.com/88313112/219373266-3f973cb0-f1be-406d-86d3-c469dfe6a0f1.png)

**Comment by omz13** [comment]
When setting text in larger sizes, traditional typographic practice was to track it slightly tighter for aesthetic reasons… therefore you might want to reconsider whether this is a bug or a deliberate feature.

**Comment by omz13** [comment]
Hi @anthonymoretti I have no idea which repo the Material Design spec is located in. Can anybody help point to the right place?

I see that Title Large and above has no tracking (IMHO it should be getting progressively tighter as the size gets larger)... but they are (correctly) progressively loosening the smaller sizes... so there's some inconsistency happening (and later on in that page they do refer to letter spacing so there is a consideration for it to be applied).

**Comment by guidezpl** [comment]
The spec was incorrect here and a fix (b/270006838) will be published in the next material.io release. Closing...
